src/app/replayer/replayer.ml

```diff
@@ -874,37 +1569,38 @@ let main ~input_file ~output_file_opt ~archive_uri ~continue_on_error () =
           "Block chain leading to target state hash does not include genesis \
            block" ;
         Core_kernel.exit 1 ) ;
-      [%log info] "Loading user command ids" ;
-      let%bind user_cmd_ids =
+      let get_command_ids (module Command_ids : Get_command_ids) name =
         match%bind
           Caqti_async.Pool.use
-            (fun db -> Sql.User_command_ids.run db target_state_hash)
+            (fun db -> Command_ids.run db target_state_hash)
             pool
         with
         | Ok ids ->
             return ids
         | Error msg ->
-            [%log error] "Error getting user command ids"
+            [%log error] "Error getting %s command ids" name
               ~metadata:[ ("error", `String (Caqti_error.show msg)) ] ;
             exit 1
       in
       [%log info] "Loading internal command ids" ;
       let%bind internal_cmd_ids =
-        match%bind
-          Caqti_async.Pool.use
-            (fun db -> Sql.Internal_command_ids.run db target_state_hash)
-            pool
-        with
-        | Ok ids ->
-            return ids
-        | Error msg ->
-            [%log error] "Error getting user command ids"
-              ~metadata:[ ("error", `String (Caqti_error.show msg)) ] ;
-            exit 1
+        get_command_ids (module Sql.Internal_command_ids) "internal"
       in
-      [%log info] "Obtained %d user command ids and %d internal command ids"
+      [%log info] "Loading user command ids" ;
+      let%bind user_cmd_ids =
+        get_command_ids (module Sql.User_command_ids) "user"
+      in
+      [%log info] "Loading Snapp command ids" ;
+      let%bind snapp_cmd_ids =
+        return []
+        (*        get_command_ids (module Sql.Snapp_command_ids) "Snapp" *)
```

Comment:
was this supposed to uncommented?

Reply:
good catch, the empty list was for testing that replaying user commands still worked